### ISSUE TITLE
Potential Issue

### ISSUE THREAD

**Issue by SirPython** [ShearOfDoom/Ethans-Bot#1]
window.setTimeout(function() {
                var messageContents = lastPostedMessages[i].messageContents;
                var messageUserName = lastPostedMessages[i].messageUserName;
        
                switch(messageContents.toLowerCase().replace(/[^a-zA-Z0-9]/g, "")) {
                    case "bot hello":
                        sendMessage("Hello " + "@" + messageUserName);
                        break;
                
                    case "bot goodbye":
                        sendMessage("Goodbye " + "@" + messageUserName);
                        break;
                }
            }, 3000);

I see a potential issue with this code. You are opening multiple threads here that will all run after 3 seconds. However, your loop will finish by then.

Basically, you loop is going to run, start a bunch of timers, and then end. When those timers end, they will call their function but `i` will be at it's final value.

Here is an example of what I mean: http://labs.codecademy.com/Cr7H#:workspace
